kernel/src/memop.rs

```diff
@@ -52,48 +52,48 @@ pub(crate) fn memop(process: &dyn Process, op_type: usize, r1: usize) -> Syscall
         // Op Type 1: SBRK
```

Comment:
The changes in this module are all backwards compatible with userspace because 2.0 assumes 32-bit words, so `u32` and `usize` are equivalent.